pkg/minikube/bootstrapper/kubeadm/util.go

```diff
@@ -122,6 +122,9 @@ func elevateKubeSystemPrivileges() error {
 	}
 	_, err = client.RbacV1beta1().ClusterRoleBindings().Create(clusterRoleBinding)
 	if err != nil {
+		if strings.Contains(err.Error(), "Client.Timeout exceeded while awaiting headers") {
```

Comment:
Hmm, there's no better way to check the type of the error rather than string matching?

Reply:
Good point, I assumed not - but checked and the error we're interested in here is an instance of `net.Error` with timeout=true. Fixed.